examples/flower-client-authentication/server.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Tuple
+
+import flwr as fl
+from flwr.common import Metrics
```

Comment:
Let's remove the entire `server_custom` example here (`server.py` is sufficient)